### ISSUE TITLE
Results By Racer page blank until first heat is run

### ISSUE THREAD

**Issue by chunter2** [jeffpiazza/derbynet#126]
When I click on Results By Racer before starting the first heat it comes up blank.  I was hoping to direct people to that page before the racing starts but I think they'll be confused if it's empty.  Once the first heat is run the page reloads with all racers and the results of the first heat.

**Comment by chunter2** [comment]
I did a little more digging on this.  Looks like the page initially comes up correctly but once all the car photos load it's then cleared.  I've attached an image of the initial and the cleared.  The initial is what I'm looking for.  
![Results By Racer 1](https://user-images.githubusercontent.com/531129/73944649-ecae7b80-48c0-11ea-905f-4099f02dd4fc.jpg)
![Results By Racer 2](https://user-images.githubusercontent.com/531129/73944662-f1732f80-48c0-11ea-8ee4-71e62efebc6d.jpg)

Thanks

**Comment by jeffpiazza** [comment]
I think we agree that the page works correctly once there are heat scheduled, so the question here is what to do about racers that aren't scheduled yet.

The current behavior of hiding unscheduled racers is intended to avoid a different issue, which is having to wait for lots and lots of empty rows to scroll by for rounds that haven't been run yet.  This is particularly irritating in the early racing rounds for a large field.

I'm inclined to leave the behavior as is, and shift the burden to the race coordinator/emcee/you to reassure the audience that the page will populate when there's something to report.

**Comment by chunter2** [comment]
Maybe I'd doing something wrong then.  I had all my racers checked in for all three dens and had scheduled all three dens but hadn't started racing yet.  The images above are from that state.  I think I agree with the unscheduled behavior.  I don't think the list should include them and would be confusing.

**Comment by jeffpiazza** [comment]
I described the current behavior incorrectly, sorry.  If a round has been scheduled but no heats have been run, there's a good chance that the schedule was constructed for one or more of the next round(s) after the current racing round.  These round results do get hidden presently, again so the audience doesn't have to see a lot of unpopulated results scroll by while the early rounds are running.

I guess I agree that the person running the race has control over when he or she generates a schedule, and so the suppression of scheduled-but-not-yet-raced results isn't really needed.

**Comment by chunter2** [comment]
So I just realized that maybe we're talking about two different pages.  Are you talking about the results by racer kiosk page or the results by race page on the initial derbynet login page?  If it's the kiosk page that autoscrolls then I think your idea of only showing running races makes sense.  If it's the other page then people at the derby's I ran would keep that page up scrolled to their kid and the times would update without scrolling or reloading the page which was nice.  They just couldn't find their kid until the first heat ran because it was blank.  Does that make sense?  I don't think I realized that it may be the same code for both pages?
